src/index.ts

```diff
@@ -117,6 +120,15 @@ export default function init(config: Knex.Config) {
 	const SeriesData = seriesData(bookshelf);
 	const WorkData = workData(bookshelf);
 
+	const kysely = new Kysely<DB>({
+		dialect: new PostgresDialect({
+			pool: new Pool(config.connection as Knex.ConnectionConfig),
```

Comment:
The config object which is loaded from our `config.json` should always have this shape with a connection property, so we can probably narrow the type which the `init` function accepts already instead of doing a type assertion here.
Knex would allow a couple of different config formats (including a connection string) which would not be compatible with the type which the pg Pool expects here.